geopandas/array.py

```diff
@@ -29,6 +29,18 @@
 from . import _vectorized as vectorized
 from .sindex import get_sindex_class
 
+try:
+    import pytest
+    import geopandas
+
+    @pytest.fixture(autouse=True)
+    def add_geopandas(doctest_namespace):
+        doctest_namespace["geopandas"] = geopandas
```

Comment:
I find it strange that it would needed to add this in normal code files. We can't add this in eg a conftest.py file?

Reply:
You're right, fixed now. I had it there at one point but something wasn't working so I reverted back, but it was likely some dumb mistake.